src/platform/testing/unit/mocha.opts

```diff
@@ -1,7 +1,8 @@
 --require src/platform/testing/unit/mocha-setup.js
 --require src/platform/testing/unit/enzyme-setup.js
 --require choma
---require @babel/polyfill
+--require core-js/stable
+--require regenerator-runtime/runtime
```

Comment:
I think we're safe to remove this file since we're on Mocha 8 now, but that's out of scope here.

Reply:
Okay cool, I will keep the file in this PR but we can look at removing it in the future.